src/main/kotlin/org/jetbrains/bio/big/Bed.kt

```diff
@@ -80,7 +82,7 @@ data class BedEntry(
         val score = when {
             fieldsNumber >= 5 && it.hasNext() -> {
                 val chunk = it.next()
-                if (chunk == ".") 0 else chunk.toShort()
+                if (chunk == ".") 0 else chunk.toInt()
```

Comment:
Let's add a test that scores > Short.MAX_VALUE could be parsed

Reply:
Added a test with score = 40000.